compiler/grainformat/format.re

```diff
@@ -1226,13 +1265,7 @@ and print_pattern =
       }
     | PPatTuple(patterns) => (
         Doc.group(
-          print_patterns(
-            ~wrapper=pat.ppat_loc,
-            ~next_loc,
-            ~comments,
-            ~original_source,
-            patterns,
-          ),
+          print_patterns(~next_loc, ~comments, ~original_source, patterns),
```

Comment:
how'd `~wrapper` get removed from this? Was that only added in an attempt to solve this problem?

Reply:
That was left over from a previous way of doing this and was unused in the code.

Is there a way to get unused variable warnings at compile time?